tests/pkg_qdsa/Makefile

```diff
@@ -0,0 +1,16 @@
+include ../Makefile.tests_common
+
+TEST_ON_CI_WHITELIST += all
+
+# qDSA is not 16 bit compatible
+BOARD_BLACKLIST := chronos msb-430 msb-430h telosb wsn430-v1_3b wsn430-v1_4 z1
+
+BOARD_INSUFFICIENT_MEMORY := arduino-duemilanove arduino-uno nucleo-f031k6
+
+CFLAGS += -DTHREAD_STACKSIZE_MAIN=\(4*THREAD_STACKSIZE_DEFAULT\)
```

Comment:
As the issue also happened in `test/tweetnacl` I think you should keep the increased stack size for avrboards

```
AVR_BOARDS := arduino-duemilanove \
               arduino-mega2560 \
               arduino-uno \
               waspmote-pro \
#
```
```
CFLAGS += -DTHREAD_STACKSIZE_MAIN=\(4*THREAD_STACKSIZE_DEFAULT+THREAD_EXTRA_STACKSIZE_PRINTF\)
```

This way, we could merge this without re-checking them for now.

Reply:
Is it also okay if I increase the `THREAD_STACKSIZE_DEFAULT` multiplier by 1? It doesn't really make sense to increase the stack size by `THREAD_EXTRA_STACKSIZE_PRINTF` if there are no printf statements in the code.

Reply:
Then it should be re-tested on `avr` which I cannot do today.
By only moving them without changing the "absurd" size, it is easy to merge. 
Fixing the stack size can be another task.

Reply:
There is only one platform where increasing it to `5*THREAD_STACKSIZE_DEFAULT` would result in a lower total stack size compared to `4*THREAD_STACKSIZE_DEFAULT+THREAD_EXTRA_STACKSIZE_PRINTF`: the lpc2387. For all other platforms, the `THREAD_STACKSIZE_DEFAULT` is larger than the `THREAD_EXTRA_STACKSIZE_PRINTF`. 

I don't mind if you want to retest on the `avr` platforms anyway, I'm not in that much of a hurry with this.

Reply:
Oh, I did not check the exact values for this. Thank you for researching it.
If 5* is bigger no need to retest for them.

So I like to keep the 5 * here as you updated and please explain it in the commit message (also that the lpc2387 is now smaller and not re-tested).
Putting your last comment directly is good for me.

Reply:
This concern and solving justification also applies for the other `LARGE_STACK_TESTS` tests you migrated.